nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Understanding internal transactions and delivery guarantees
+reviewed: 2016-04-20
+tags:
+- Azure
+- Cloud
+- Transactions
+---
+
+## Understanding internal transactions and delivery guarantees
+
+### Versions 6 and below
```

Comment:
I'll look into partials, wasn't aware this existed (until an hour ago when a partial broke :))